src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  padding: 2px;
+  background-repeat: no-repeat;
+}
+
+.stars .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(n+1) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
All good, but you can make it shorter by using , like this:
.stars--3 .stars__star:nth-child(n+3),
.stars--4 .stars__star:nth-child(n+2) {
  background-image: url(./images/star-active.svg);
}

It will be same but shorter)

Reply:
thank you, I'll fix it )